kie-server-parent/kie-server-services/kie-server-services-common/src/main/resources/META-INF/services/org.kie.server.api.security.SecurityAdapter

```diff
@@ -1,4 +1,5 @@
 org.kie.server.services.impl.security.adapters.TomcatSecurityAdapter
 org.kie.server.services.impl.security.adapters.JMSSecurityAdapter
```

Comment:
BrokerSecurityAdapter

and also change `JmsUserGroupAdapter` to `BrokerUserGroupAdapter` in:
kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/resources/META-INF/services/org.jbpm.services.task.identity.adapter.UserGroupAdapter

Reply:
correct

Reply:
fixed with the latest changes